relay-event-normalization/src/normalize/mod.rs

```diff
@@ -346,6 +346,9 @@ pub struct PerformanceScoreWeightedComponent {
     pub p10: f64,
     /// Median used to define the log-normal for calculation
     pub p50: f64,
+    // Whether the measurement is optional. If the measurement is missing, performance score processing
+    // may still continue, and the weight will be 0.
+    pub optional: Option<bool>,
```

Comment:
If there is no difference between `None` and `false`:

```rs
#[serde(default)]
pub optional: bool,
```

Reply:
Thanks! There's no difference so i've updated to using this